tests/test_definitions/multicolumn_map_expectations/expect_select_column_values_to_be_unique_within_record.json

```diff
@@ -126,26 +126,19 @@
       },
       "out": {
         "unexpected_list": [{"a": 1.0, "b": 1.0}, {"a": 1.0, "b": 1.0}, {"a": 1.0, "b": 1.0}, {"a": 2.0, "b": 2.0}, {"a": 2.0, "b": 2.0}],
-        "unexpected_index_list": [0,2,4,5,7],
+        "unexpected_index_list": [{"a": 1.0, "pk_index": 0, "b": 1.0}, {"a": 1.0, "pk_index": 2, "b": 1.0}, {"a": 1.0, "pk_index": 4, "b": 1.0}, {"a": 2.0, "pk_index": 5, "b": 2.0}, {"a": 2.0, "pk_index": 7, "b": 2.0}],
         "success": false
       }
     },{
       "title": "unexpected_values_exact_match_out_with_index_list",
-      "exact_match_out" : true,
```

Comment:
Why was this changed to false?

Reply:
This is because the JSON tests handle both v2 and v3 expectations. 
* `exact_match_out` ensures the entire dictionary matches expectations, but this can't be the case. 
* V3 will contain ID/PK related parameters in the `unexpected_index_list` while V2 will not